ProefProeveVeluwe/Assets/Scripts/Runtime/GPS/GPSDebug.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Runtime
+{
+    /// <summary>
+    /// Gets the current GPS values of the Device.
+    /// </summary>
+    public class GPSDebug : MonoBehaviour
+    {
+        [SerializeField] private Text GPSStatus;
+        [SerializeField] private Text latitudeValue;
+        [SerializeField] private Text longitudeValue;
+        [SerializeField] private Text altitudeValue;
+        [SerializeField] private Text horizontalAccuracyValue;
+        [SerializeField] private Text timeStampValue;
+
+        private GPS _GPSData;
```

Comment:
Even thou GPS is writen with capital letters naming conventions require this to me pascalcase